qmt/geometry/freecad/fileIO.py

```diff
@@ -13,18 +13,33 @@
 from .auxiliary import silent_stdout
 
 
-def exportMeshed(obj, fileName):
-    """ Export a mesh of an object to the given file name.
+def exportMeshed(obj_list, file_name):
```

Comment:
You might consider using type hints instead of writing the type in the docstring:
```
from typing import List
...
def exportMesh(obj_list: List, file_name: str):
```
This doesn't enforce the types at runtime so you'll still need the type check later, but it's useful for the user/programmer.

Also does `obj_list` have to be a list? Or will tuples do as well? If so you can use `Sequence` instead of `List`.